apps/website-25/src/components/careers/CareersCallout.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { CTAButton } from '@bluedot/ui';
+
+const Callout: React.FC = () => {
+  const title = 'Join us in our mission to ensure humanity safely navigates the transition to transformative AI.';
+  const ctaLabel = 'View our careers';
+  const ctaLink = '/careers';
+
+  return (
+    <div className="callout bg-[radial-gradient(ellipse_at_30%,_#fff_0%,_#6687ff_100%)] rounded-lg p-8 m-8 flex flex-col items-center">
```

Comment:
lets increase the y padding to get slightly closer to the mocks:
```suggestion
    <div className="callout bg-[radial-gradient(ellipse_at_30%,_#fff_0%,_#6687ff_100%)] rounded-lg py-16 px-8 m-8 flex flex-col items-center">
```